Controller/Adminhtml/Import/Ajax.php

```diff
@@ -97,6 +85,18 @@ public function execute()
                     $result['success']  = $client->productBatch($serializedProducts);
                     break;
                 case 'orders':
+                    
+                    if ($chunkId == 0) {
```

Comment:
this looks weird, can the deleted products be moved to separate stage between `products` and `orders`?

Reply:
Will implement new import step with the next commit.